android/app/src/main/res/drawable/background_splash.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@color/blue"/>
+    <item
+        android:width="200dp"
+        android:height="200dp"
+        android:drawable="@mipmap/icon"
```

Comment:
Here where you declare the image you don't have an image with that name on your mipmap directory
maybe you need to change `android:drawable="@mipmap/icon"` for  `android:drawable="@mipmap/splash_icon"`